packages/blob/package.json

```diff
@@ -27,14 +27,6 @@
     "undici": "./dist/undici-browser.js",
     "crypto": "./dist/crypto-browser.js"
   },
-  "types": "./dist/index.d.ts",
-  "typesVersions": {
```

Comment:
I've removed `typesVersions` when rebasing this on top of main, but this led to an error: https://github.com/vercel/storage/actions/runs/6221448244/job/16883410004?pr=314#step:8:245

Does `typesVersions` need to stay?

Reply:
Brought this back in https://github.com/lukahartwig/storage/commit/728fd7ddb47ec0bd1c59a785a6ea0333bb3e07db

Reply:
I think it's trying to import `@vercel/blob/client` from a CJS context. Typescript is resolving this to `./packages/blob/client.d.ts`. A dirty hack would be to add a file there that reexports the types from the dist folder.

If you don't find a solution until then I can look at it this evening.

Reply:
It works when I bring back `typesVersions`. It's not so obvious in this PR as it fails due to you being an outside contributor, but I recreated your changes in https://github.com/vercel/storage/pull/391 and it's all green there with `typesVersions` present.

Not sure if I prefer a hack over declaring `typesVersions`, but I'm keen to hear your take on it

Reply:
Both are hacks, I'm fine with the `typesVersion` solution. Seem more maintainable as well 👍

Reply:
@dferber90 There was a file at some point but the typesVersion solution is less invading code wise. i.e. like an ie6 fix I prefer for it to be hidden as much as possible

Reply:
Also the file solution is not complete, you actually need a whole folder for it to work, see https://github.com/vercel/storage/commit/b8b76332112c124c68ef4b240f9678e05c84da52 and https://antfu.me/posts/types-for-sub-modules